include/platform/2_0.hpp

```diff
@@ -15,6 +15,13 @@ enum ButtonID {
     R_Left,
     R_Right,
     MaxID,
+
+    // Shadow ids for backwards compatibility with 1.x examples
+    Enter = 0,
+    Up = 1,
+    Down = 2,
+    Left = 3,
+    Right = 4,
```

Comment:
Možná nechceme? Asi můžu upravit ty examply, ale je to skoro všude...

Reply:
Klidně bych to nechal, dává to za mě smysl... možná bych je označil deprecated tagem